packages/frontend/src/features/sqlRunner/transformers/TableDataTransformer.ts

```diff
@@ -4,30 +4,31 @@ import { getRawValueCell } from '../../../hooks/useColumns';
 import { type useSqlQueryRun } from '../hooks/useSqlQueryRun';
 
 // TODO: Move to the common package
-export type TableChartSqlConfig =
-    | {
-          columns: Record<
-              string,
-              {
-                  visible: boolean;
-                  reference: string;
-                  label: string;
-                  frozen: boolean;
-                  order?: number;
-              }
-          >;
-      }
-    | undefined;
+export type TableChartSqlConfig = {
```

Comment:
I put the bar chart type in packages/common/src/types/visualizations.ts

Reply:
Sounds good!

Reply:
Done here: https://github.com/lightdash/lightdash/pull/10663/commits/a6beebcfcc7fa2fea6064b91e0dfc8410a2ea92e
didn't remove the word `Sql` because there's already a TableChartConfig
we should revisit this after being able to save charts IMO